ansible/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+inventory = inventory.yaml
```

Comment:
`pipelining = True` is useful for performance boost, see https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-pipelining

(ssh multiplexing in local sshconfig is also worth having)